cmd/mtaBuild_test.go

```diff
@@ -1,360 +0,0 @@
-		}
-		assert.Equal(t, "myName.mtar", cpe.mtarFilePath)
-	})
-
-	t.Run("Settings file releatd tests", func(t *testing.T) {
-
-		var settingsFile string
-		var settingsFileType maven.SettingsFileType
-
-		defer func() {
-			getSettingsFile = maven.GetSettingsFile
-		}()
-
-		getSettingsFile = func(
-			sfType maven.SettingsFileType,
-			src string,
-			fileUtilsMock piperutils.FileUtils,
-			httpClientMock piperhttp.Downloader) error {
-			settingsFile = src
-			settingsFileType = sfType
-			return nil
-		}
-
-		fileUtils := MtaTestFileUtilsMock{}
-		fileUtils.existingFiles = make(map[string]string)
-		fileUtils.existingFiles["package.json"] = "{\"name\": \"myName\", \"version\": \"1.2.3\"}"
-
-		t.Run("Copy global settings file", func(t *testing.T) {
-
-			defer func() {
-				settingsFile = ""
-				settingsFileType = -1
-			}()
-
-			e := mock.ExecMockRunner{}
-
-			options := mtaBuildOptions{ApplicationName: "myApp", GlobalSettingsFile: "/opt/maven/settings.xml", MtaBuildTool: "cloudMbt", Platform: "CF", MtarName: "myName"}
-
-			err := runMtaBuild(options, &cpe, &e, &fileUtils, &httpClient)
-
-			assert.Nil(t, err)
-
-			assert.Equal(t, settingsFile, "/opt/maven/settings.xml")
-			assert.Equal(t, settingsFileType, maven.GlobalSettingsFile)
-		})
-
-		t.Run("Copy project settings file", func(t *testing.T) {
-
-			defer func() {
-				settingsFile = ""
-				settingsFileType = -1
-			}()
-
-			e := mock.ExecMockRunner{}
-
-			options := mtaBuildOptions{ApplicationName: "myApp", ProjectSettingsFile: "/my/project/settings.xml", MtaBuildTool: "cloudMbt", Platform: "CF", MtarName: "myName"}
-
-			err := runMtaBuild(options, &cpe, &e, &fileUtils, &httpClient)
-
-			assert.Nil(t, err)
-
-			assert.Equal(t, "/my/project/settings.xml", settingsFile)
-			assert.Equal(t, maven.ProjectSettingsFile, settingsFileType)
-		})
-	})
-}
-
-type MtaTestFileUtilsMock struct {
-	existingFiles map[string]string
-	writtenFiles  map[string]string
-	copiedFiles   map[string]string
-}
-
-func (f *MtaTestFileUtilsMock) FileExists(path string) (bool, error) {
-
-	if _, ok := f.existingFiles[path]; ok {
-		return true, nil
-	}
-	return false, nil
-}
-
-func (f *MtaTestFileUtilsMock) Copy(src, dest string) (int64, error) {
-
-	if f.copiedFiles == nil {
-		f.copiedFiles = make(map[string]string)
-	}
-	f.copiedFiles[src] = dest
-
-	return 0, nil
-}
-
-func (f *MtaTestFileUtilsMock) FileRead(path string) ([]byte, error) {
-	return []byte(f.existingFiles[path]), nil
-}
-
-func (f *MtaTestFileUtilsMock) FileWrite(path string, content []byte, perm os.FileMode) error {
-
-	if f.writtenFiles == nil {
-		f.writtenFiles = make(map[string]string)
-	}
-
-	if _, ok := f.writtenFiles[path]; ok {
-		delete(f.writtenFiles, path)
-	}
-	f.writtenFiles[path] = string(content)
-	return nil
-}
-
-func (f *MtaTestFileUtilsMock) MkdirAll(path string, perm os.FileMode) error {
-	return nil
-}
```

Comment:
Why delete this entire file? Because it will be restored (and fixed) in the other PR?

Reply:
will be restored in follow up pr by @KevinHudemann